src/app/content/components/TableOfContents/styled/index.tsx

```diff
@@ -148,6 +150,10 @@ export const NavOl = styled.ol<{section: ArchiveTree}>`
       }
     `;
   }}
+
+  ${theme.breakpoints.mobile(css`
+    padding-bottom: ${navOlPaddingBottom}rem;
```

Comment:
this is too agressive, it'll put it at the bottom of every chapter element, you want it only in the scrollable one, here https://github.com/openstax/rex-web/pull/317/files#diff-95a8ce6db53e1d02036b3a9371a5e9bbR67